script.js

```
@@ -2086,3 +2233,336 @@ function loadFavorites() {
+    
+    showSuccessMessage(t('eventSaved'));
+}
+
+// Delete event
+function deleteEvent() {
+    if (!editingActivity) return;
+    
+    if (!confirm(t('confirmDelete'))) return;
+    
+    if (editingActivity.id && editingActivity.id.startsWith('custom_')) {
+        // Delete custom activity
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)